src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -115,7 +115,7 @@ HMODULE LoadCoreClr(dnx::trace_writer& trace_writer)
     }
     else
     {
-        coreclr_module = LoadLibraryExW(L"coreclr.dll", NULL, 0);
+        coreclr_module = LoadLibraryExW(dnx::utils::path_combine(GetModuleDirectory(nullptr), L"coreclr.dll").c_str(), NULL, 0);
```

Comment:
Yes, there is one more place where we do it. I can refactor the code to have just one call and pass the parameter around if this is a (performance/startup time?) concern.